src/coreclr/vm/comdelegate.cpp

```diff
@@ -1804,6 +1811,19 @@ extern "C" void QCALLTYPE Delegate_Construct(QCall::ObjectHandleOnStack _this, Q
     END_QCALL;
 }
 
+MethodDesc *COMDelegate::GetMethodDescForOpenVirtualDelegate(OBJECTREF orDelegate)
```

Comment:
Can you please also update `COMDelegate::GetMethodDesc()` below to call this API. I'm assuming it is appropriate in this case.

Reply:
Ok, good idea.